src/core_codemods/sonar/sonar_jwt_decode_verify.py

```diff
@@ -0,0 +1,43 @@
+import libcst as cst
+from codemodder.codemods.base_codemod import Reference
+from codemodder.result import same_line, fuzzy_column_match
+from codemodder.codemods.sonar import SonarCodemod
+from codemodder.codemods.libcst_transformer import (
+    LibcstTransformerPipeline,
+)
+from core_codemods.jwt_decode_verify import JwtDecodeVerify, JwtDecodeVerifyTransformer
+
+
+class JwtDecodeVerifySonarTransformer(JwtDecodeVerifyTransformer):
+    def filter_by_result(self, node) -> bool:
+        """
+        Special case result-matching for this rule because the sonar
+        results returned have a start/end column for the verify keyword
+        within the `decode` call, not for the entire call like semgrep returns.
+        """
+        if self.results is None:
+            return False
+        match node:
+            case cst.Call():
+                pos_to_match = self.node_position(node)
+                return any(
+                    self.match_location(pos_to_match, result) for result in self.results
+                )
+        return False
+
+    def match_location(self, pos, result):
+        return any(
+            same_line(pos, location) and fuzzy_column_match(pos, location)
+            for location in result.locations
+        )
```

Comment:
Did you encounter any examples where `fuzzy_column_match` is needed? From my observations most nodes reported by sonar would match libcst ones with one exception: `Tuple`s. If you look at the `match_location` from `SonarResult` you can see the exception treated there. We should discuss how to treat location match at some point.

Reply:
You're going to have to give me an example because adding `fuzzy_column_match`  is the entire point of this PR. As I stated in the PR description, sonar returns index for the keyword=value, while we need jwt.decode(...keyword=value)

Reply:
In the simplest terms, does anything break if you remove `fuzzy_column_match` and replace it by exact match?
If so, does it break consistently? (e.g. always with the same type of node)

Reply:
Yes, there are no location matches without fuzzy matching.